R/pkg/R/sparkR.R

```diff
@@ -381,6 +381,10 @@ sparkR.session <- function(
     deployMode <- sparkConfigMap[["spark.submit.deployMode"]]
   }
 
+  if (!exists("spark.sql.default.derby.dir", envir = sparkConfigMap)) {
```

Comment:
I see. This PR is just to fix the default behavior for releasing SparkR on CRAN. 

We need to add code comments to explain what it is for the future code maintainer/reader.